astropy/table/tests/test_row.py

```diff
@@ -343,7 +348,8 @@ def test_row_tuple_column_slice_transaction():
     assert t[1] == tc[1]
 
 
-def test_uint_indexing():
+@pytest.mark.parametrize("dtype", [int, np.int_, np.uint])
+def test_uint_indexing(dtype, small_table):
```

Comment:
I'm not convinced there is a substantial performance improvement in the end due to overhead in collecting three pytests tests instead of one and using a fixture. Probably better to just revert this unrelated change.

Reply:
I think that parametrizing this test would make the output of `pytest` more user-friendly, but having changed the implementation of the `Row.get()` unit tests I can't justify making this change in this pull request.